docs/code-templates/TwitchLib/FDGTConnect.md

```diff
@@ -0,0 +1,25 @@
+```csharp
+// Additional package to download
+// - TwitchLib.Communication.Clients.FDGTClient 
```

Comment:
Will this be enough for users to understand what to do? I'm not familiar with the C# ecosystem, so I have no idea. 😬

Reply:
I guess I could add a little more text to it. ^^

Reply:
Don't feel obligated, my question is really this: If a C# user comes to the `fdgt` documentation and reads this comment, will they know where to download it from this additional package? Will they know how to download it (package manager, etc)? Is there additional configuration needed?

If there is general C# knowledge that will allow a typical user to understand this comment, then I'm good with it. If not, let's bulk it out to ensure that the typical C# user will understand. 😁

Reply:
Yeah, most people know how to download NuGet-packages at least. It's like yarn/npm. ^^